lib/engines/hcktest/drivers/jtd.json

```diff
@@ -0,0 +1,28 @@
+{
+  "definitions": {
+    "command_info": {
+      "properties": {
+        "desc": { "type": "string" },
+        "run": { "type": "string" }
+      }
+    }
+  },
```

Comment:
It's fine but I wonder why you make the use of definition here.

Reply:
Just copy-paste from the platform.  Do you want to remove this?

Reply:
It makes sense. Let's leave it.